diff --git a/node_modules/react-modal-sheet/src/sheet.tsx b/node_modules/react-modal-sheet/src/sheet.tsx
index b1ac36b..e1820e2 100644
--- a/node_modules/react-modal-sheet/src/sheet.tsx
+++ b/node_modules/react-modal-sheet/src/sheet.tsx
@@ -161,7 +161,7 @@ const Sheet = forwardRef<any, SheetProps>(
         onClose();
       } else {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        const sheetHeight = sheetRef.current!.getBoundingClientRect().height;
+        const sheetHeight = sheetRef?.current!.getBoundingClientRect().height;
         const currentY = y.get();
 
         let snapTo = 0;
@@ -214,10 +214,10 @@ const Sheet = forwardRef<any, SheetProps>(
     useImperativeHandle(ref, () => ({
       y,
       snapTo: (snapIndex: number) => {
-        const sheetEl = sheetRef.current;
+        const sheetEl = sheetRef?.current;
 
         if (snapPoints?.[snapIndex] !== undefined && sheetEl) {
-          const sheetHeight = sheetEl.getBoundingClientRect().height;
+          const sheetHeight = sheetEl?.getBoundingClientRect().height;
           const snapPoint = snapPoints[snapIndex];
           const snapTo = validateSnapTo({
             snapTo: sheetHeight - snapPoint,
